examples/machine/DeePMD-kit-1.x/machine-ali.json

```diff
@@ -4,7 +4,7 @@
       "machine": {
         "batch": "shell",
         "hostname": "",
-        "password": "975481DING!",
+        "password": "PASSWORD",
```

Comment:
Maybe not related to this PR - I would suggest to use [`ssh-copy-id`](https://www.ssh.com/academy/ssh/copy-id) instead of using `password`。